src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,21 @@
+package duke.command;
+
+import duke.io.Layout;
+import duke.io.Parser;
+import duke.task.Task;
+
+import java.util.ArrayList;
+
+public abstract class Command {
+    
+    protected ArrayList<Task> tasks;
```

Comment:
Is tasks really needed for all commands? I am referring to commands like help. I would suggest to add tasks as a field in classes that inherit from Command and initialise it with a constructor in the class.